src/main/java/visualizations/ScatterChart.java

```diff
@@ -23,14 +23,14 @@
     }
 
     /**
-     * Plots a scatter chart given the chart title,
+     * Plots a scatter chart given the chart options,
      * data set in Table format,
```

Comment:
I can't see ```Table``` argument. The fact that the function uses ```Table``` is irrelevant for the client code. Such things typically we should document them whenever there is some very good reason to do so for example thread safety, memory consumption compute time etc

Reply:
I see your point and corrected the documentation.